packages/example-codehub/test/support/util.ts

```diff
@@ -4,22 +4,24 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import {supertest} from '@loopback/testlab';
-import {CoreBindings} from '@loopback/core';
+import {RestBindings} from '@loopback/rest';
 import {CodeHubApplication} from '../../src/codehub-application';
+import { RestServer } from '../../../rest/src/rest-server';
 
 export async function createClientForApp(app: CodeHubApplication) {
   const url = (await app.info()).url;
   return supertest(url);
 }
 
-export function createApp() {
+export async function createApp() {
   const app = new CodeHubApplication();
-  app.bind(CoreBindings.HTTP_PORT).to(0);
+  const server = await app.get('servers.RestServer') as RestServer;
+  server.bind(RestBindings.HTTP_PORT).to(0);
```

Comment:
I think it should be still possible to bind the HTTP port at application level. Maybe `RestBindings.HTTP_PORT` should use a custom prefix to avoid clash with other HTTP-based transports?

Also, unless the rest server is going to support dual listener for both HTTP and HTTPS, then I think the `HTTP_` prefix no longer makes sense and `RestBindings.PORT` would be a better name.

```ts
app.bind(RestBindings.PORT).to(0);
// 'rest.port'

app.bind(GrpcBindings.PORT).to(0);
// 'grpc.port'
```

Thoughts?

Reply:
My thoughts on keeping `HTTP_PORT` were largely surrounding the idea of a parallel `HTTPS_PORT`, but we don't have anything like that implemented. @raymondfeng and I were talking about the Server implementations potentially accepting actual Node.js server instances (`httpServer`, `httpsServer`, `http2Server`, etc).

In that sense, it's no longer important to distinguish the protocol type for the port binding, so I'll change it. :)